### PR TITLE
Update eddie config docs & added viralrecon config

### PR DESCRIPTION
**Pull request by ameynert**
Addition of viralrecon config.
Update of eddie config documentation.

### CODE REVIEW

File: conf/pipeline/viralrecon/eddie.config [hunk 1]
```diff
@@ -0,0 +1,9 @@
+process {
+  withName : '.*PICARD.*' {
```
Comment (drpatelh) [inline conf/pipeline/viralrecon/eddie.config:2]: You will also need to edit this file https://github.com/nf-core/configs/blob/master/pipeline/viralrecon.config loading the `eddie` profile like [here](https://github.com/nf-core/configs/blob/47217cae5b2286b7a288ed1bd4616990d5de7a26/pipeline/rnaseq.config#L12)

Comment (drpatelh) [reply conf/pipeline/viralrecon/eddie.config:2]: Be great if you can test this via your fork to see if it works. Should just need to change [this](https://github.com/nf-core/viralrecon/blob/f0171324a60d1759ca7f5d02351372d97d22cc12/nextflow.config#L106) parameter.

Comment (ameynert) [reply conf/pipeline/viralrecon/eddie.config:2]: > You will also need to edit this file https://github.com/nf-core/configs/blob/master/pipeline/viralrecon.config loading the `eddie` profile like [here](https://github.com/nf-core/configs/blob/47217cae5b2286b7a288ed1bd4616990d5de7a26/pipeline/rnaseq.config#L12)

Done.

File: pipeline/viralrecon.config [hunk 2]
```diff
@@ -5,3 +5,4 @@
  */
 
 includeConfig "${params.custom_config_base}/conf/pipeline/viralrecon/genomes.config"
+includeConfig "${params.custom_config_base}/conf/pipeline/viralrecon/eddie.config"
```
Comment (drpatelh) [inline pipeline/viralrecon.config:8]:
```suggestion

profiles {
  eddie { includeConfig "${params.custom_config_base}/conf/pipeline/viralrecon/eddie.config" }
}
```

Comment (drpatelh) [reply pipeline/viralrecon.config]: The `genomes.config` always needs to be loaded when running the pipeline which is why it is outside of the `profiles` block.

**Review by drpatelh — APPROVED** [review]

**Pull request merged**